### PR TITLE
added actions rocket controller

### PR DESCRIPTION
**Pull request by Zeegeuze**
Thanks to check changes in actions rocket controller.

### CODE REVIEW

File: app/controllers/rockets_controller.rb [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+    @rocket = Rocket.new
   end
 
   def index
+    @rockets = Rocket.all
   end
 
   def create
+    @rocket = Rocket.new(rocket_params)
+    @rocket.save
```
Comment (nicdebray) [inline app/controllers/rockets_controller.rb]: what about the 
if @rocket.save
redirect_to ...
else render: new
end

File: app/controllers/rockets_controller.rb [hunk 2]
```diff
@@ -15,8 +23,23 @@ def edit
   end
 
   def update
+    @rocket.update(rocket_params)
+    redirect_to rocket_path(@rocket)
```
Comment (nicdebray) [inline app/controllers/rockets_controller.rb]: it is rockets_path  with "s"

File: app/controllers/rockets_controller.rb [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
   def index
+    @rockets = Rocket.all
   end
 
   def create
+    @rocket = Rocket.new(rocket_params)
+    @rocket.save
+
+    # no need for app/views/restaurants/create.html.erb
+    redirect_to rocket_path(@rocket)
```
Comment (nicdebray) [inline app/controllers/rockets_controller.rb]: it is rockets_path  with "s"

File: app/controllers/rockets_controller.rb (hunk 1 above)
Comment (Zeegeuze) [reply app/controllers/rockets_controller.rb]: Done

File: app/controllers/rockets_controller.rb (hunk 2 above)
Comment (Zeegeuze) [reply app/controllers/rockets_controller.rb]: corrected

File: app/controllers/rockets_controller.rb (hunk 3 above)
Comment (Zeegeuze) [reply app/controllers/rockets_controller.rb]: Done

**Pull request merged**